types/react/global.d.ts

```diff
@@ -143,6 +143,199 @@ interface SVGTSpanElement extends SVGElement { }
+    "td": HTMLTableDataCellElement;
+    "template": HTMLTemplateElement;
+    "textarea": HTMLTextAreaElement;
+    "tfoot": HTMLTableSectionElement;
+    "th": HTMLTableHeaderCellElement;
+    "thead": HTMLTableSectionElement;
+    // "time": HTMLTimeElement;
+    "title": HTMLTitleElement;
+    "tr": HTMLTableRowElement;
+    "track": HTMLTrackElement;
+    "u": HTMLElement;
+    "ul": HTMLUListElement;
+    "var": HTMLElement;
+    "video": HTMLVideoElement;
+    "wbr": HTMLElement;
+}
+
+interface HTMLElementDeprecatedTagNameMap {
+    "listing": HTMLPreElement;
+    "xmp": HTMLPreElement;
+}
+
+interface SVGElementTagNameMap {
+    // "a": SVGAElement;
+    "circle": SVGCircleElement;
+    "clipPath": SVGClipPathElement;
+    "defs": SVGDefsElement;
+    "desc": SVGDescElement;
+    "ellipse": SVGEllipseElement;
+    "feBlend": SVGFEBlendElement;
+    "feColorMatrix": SVGFEColorMatrixElement;
+    "feComponentTransfer": SVGFEComponentTransferElement;
+    "feComposite": SVGFECompositeElement;
+    "feConvolveMatrix": SVGFEConvolveMatrixElement;
+    "feDiffuseLighting": SVGFEDiffuseLightingElement;
+    "feDisplacementMap": SVGFEDisplacementMapElement;
+    "feDistantLight": SVGFEDistantLightElement;
+    "feFlood": SVGFEFloodElement;
+    "feFuncA": SVGFEFuncAElement;
+    "feFuncB": SVGFEFuncBElement;
+    "feFuncG": SVGFEFuncGElement;
+    "feFuncR": SVGFEFuncRElement;
+    "feGaussianBlur": SVGFEGaussianBlurElement;
+    "feImage": SVGFEImageElement;
+    "feMerge": SVGFEMergeElement;
+    "feMergeNode": SVGFEMergeNodeElement;
+    "feMorphology": SVGFEMorphologyElement;
+    "feOffset": SVGFEOffsetElement;
+    "fePointLight": SVGFEPointLightElement;
+    "feSpecularLighting": SVGFESpecularLightingElement;
+    "feSpotLight": SVGFESpotLightElement;
+    "feTile": SVGFETileElement;
+    "feTurbulence": SVGFETurbulenceElement;
+    "filter": SVGFilterElement;
+    "foreignObject": SVGForeignObjectElement;
+    "g": SVGGElement;
+    "image": SVGImageElement;
+    "line": SVGLineElement;
+    "linearGradient": SVGLinearGradientElement;
+    "marker": SVGMarkerElement;
+    "mask": SVGMaskElement;
+    "metadata": SVGMetadataElement;
+    "path": SVGPathElement;
+    "pattern": SVGPatternElement;
+    "polygon": SVGPolygonElement;
+    "polyline": SVGPolylineElement;
+    "radialGradient": SVGRadialGradientElement;
+    "rect": SVGRectElement;
+    // "script": SVGScriptElement;
+    "stop": SVGStopElement;
+    // "style": SVGStyleElement;
+    "svg": SVGSVGElement;
+    "switch": SVGSwitchElement;
+    "symbol": SVGSymbolElement;
+    "text": SVGTextElement;
+    "textPath": SVGTextPathElement;
+    // "title": SVGTitleElement;
+    "tspan": SVGTSpanElement;
+    "use": SVGUseElement;
+    "view": SVGViewElement;
+}
+
+/**
+ * In lib.dom.d.ts this type alias is deprecated
+ */
+type ElementTagNameMap = HTMLElementTagNameMap & Pick<SVGElementTagNameMap, Exclude<keyof SVGElementTagNameMap, keyof HTMLElementTagNameMap>>;
```

Comment:
Copying these from `lib.dom.d.ts` so `ElementRef` can safely be used in react-native projects. Open to better alternatives.